src/less/combobox/base/combobox.less

```diff
@@ -1,40 +1,77 @@
+
 .combobox {
     box-sizing: border-box;
     display: inline-block;
 }
 
 .combobox__control > input {
     appearance: none;
+    background-color: @combobox-input-background-color;
     border: 1px solid;
+    border-color: @combobox-input-border-color;
+    border-radius: @combobox-input-border-radius;
     box-sizing: border-box;
     font-family: inherit;
+    font-size: @combobox-input-font-size;
+    height: @combobox-input-height;
+    padding: @combobox-input-padding;
     text-align: left;
 
+    &:focus {
+        border-color: @combobox-input-focus-border-color;
+        outline: 0;
+    }
+
     &[readonly] {
         // fixes the cursor in firefox (1 of 2)
         color: transparent;
         cursor: default;
+        // fixes the cursor in firefox (2 of 2)
+        text-shadow: 0 0 0 @combobox-input-readonly-text-shadow-color;
         -webkit-user-select: none; // Fix to get rid of unintended cursor in iOS mobile
+
+        &::-moz-selection {
```

Comment:
Can combine with ::selection? See the comment on the variable section

Reply:
👍